solutions/src/queries/getData.js

```diff
@@ -0,0 +1,16 @@
+// Add code below to query your database
+const dbConnection = require('../database/db_connection.js');
+
+const getData = (cb) =>{
+    dbConnection.query(`SELECT * FROM users`
+    , (err, res) => {
+    if {
+      (err) cb(err);
```

Comment:
the condition for the if is inside, this line does not run. 
`if (err)` instead :flushed: